charts/csi-cloudscale/values.yaml

```diff
@@ -0,0 +1,66 @@
+cloudscale:
+  apiUrl: https://api.cloudscale.ch/
+  token:
+    value: "secret"
+    # existingSecret:
+
+installCRDs: false
+
+csi:
+  allowVolumeExpansion: true
+  storageClasses:
+    - name: cloudscale-volume-ssd
+      volumeType: ssd
+      default: true
+      luks:
+        enabled: false
+    - name: cloudscale-volume-ssd-luks
+      volumeType: ssd
+      luks:
+        enabled: true
+        cipher: aes-xts-plain64
+        keySize: 512
+    - name: cloudscale-volume-bulk
+      volumeType: bulk
+      luks:
+        enabled: false
+    - name: cloudscale-volume-bulk-luks
+      volumeType: bulk
+      luks:
+        enabled: true
+        cipher: aes-xts-plain64
+        keySize: 512
+
+controller:
+  replicas: 1
+
+provisioner:
+  name: csi.cloudscale.ch
+  image:
+    repository: quay.io/k8scsi/csi-provisioner
+    tag: v2.0.4
+    pullPolicy: IfNotPresent
+
+attacher:
+  image:
+    repository: quay.io/k8scsi/csi-attacher
+    tag: v3.0.2
+    pullPolicy: IfNotPresent
+
+resizer:
+  image:
+    repository: quay.io/k8scsi/csi-resizer
+    tag: v1.0.1
+    pullPolicy: IfNotPresent
+
+plugin:
+  image:
+    repository: cloudscalech/cloudscale-csi-plugin
+    tag: v3.2.0
+    pullPolicy: IfNotPresent
```

Comment:
Currently set to Always.

Reply:
Is there any reason for this?

Reply:
I _think_ it has to do with the `:dev` version of the image. We used this tag to test new builds.

Reply:
I'm probably to blame for that and Alain is right that this has to do with the `dev` version of the image.

Reply:
`ifNotPresent` is afaik best practise

Reply:
When we switch to helm, we can use `IfNotPresent` for the stable releases and override it when debugging/developing.